CONTRIBUTING.adoc

```diff
@@ -86,13 +90,10 @@ export PATH=$PATH:$GOPATH/bin
         $ cd _output/local/bin/linux/amd64
         $ sudo ./openshift start
 
-+
-NOTE: Replace "linux/amd64" with the appropriate value for your platform/architecture.
-
 10.  Launch another terminal, change into the same directory you started OpenShift, and deploy the private docker registry within OpenShift with the following commands (note, the --credentials option allows secure communication between the internal OpenShift Docker registry and the OpenShift server, and the --config option provides your identity (in this case, cluster-admin) to the OpenShift server):
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
-        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
+        $ sudo chmod a+rw openshift.local.config/master/admin.kubeconfig
```

Comment:
@liggitt I don't think we are in the habit of having admin permissions setup on the inst images by default.  We have been toying with the idea of a bootstrap image that does do that sort of thing but it hasn't really been used or completed.